scripts/benchmarks/scenarios.yaml

```diff
@@ -110,8 +112,8 @@ scenarios:
             - accelerated-peft-autogptq-foak
         arguments:
             learning_rate: 2e-4
-            bf16: True
-            torch_dtype: bfloat16
+            fp16: True
```

Comment:
you can put even a link to this issue in comments https://github.com/foundation-model-stack/fms-acceleration/issues/84